src/docs/apiary.apib

```diff
@@ -1,9 +1,29 @@
-# Data-manager
+FORMAT: 1A
+HOST: http://localhost:3000
+
+# Data Management API
+
+This document describes the APIs using DataManager as a standalone process. If used with
+other dojot's components, the endpoints might be changed a bit. Check [dojot's
+documentation](http://dojotdocs.readthedocs.io/) to check all of them.
+
+
+# Import / Export
+Before describing the DataManager APIs, we must explain the import and export functionality. 
+When you export data from Dojot, a JSON file is generated for you. That JSON file contains
+all current templates, devices, flows and remote nodes in your environment. You can save this file
+to import this data later. The import functionality uses this file to regenerate data
+inside Dojot. However, when you import that file, then the templates, devices, flows and remote nodes will
+be created again. It's important to know that this operation is destructive. This means that, 
+if you have templates, devices, flows or remote nodes in your environment, all of them will be
+deleted during the import process, before they are generated. So, be careful. 
```

Comment:
Maybe point out that this is valid for a particular tenant (and not all of them)?

Reply:
That sounds nice!